components/Migrate-CPTUI/Migrate-CPTUI.php

```diff
@@ -32,9 +54,14 @@ class Pods_Migrate_CPTUI extends PodsComponent {
      *
      * @since 2.0
      */
-    public function __construct () {
-        $this->post_types = (array) get_option( 'cpt_custom_post_types', array() );
-        $this->taxonomies = (array) get_option( 'cpt_custom_tax_types', array() );
+    public function __construct() {
+
+        $this->post_option_name = $this->get_option_name( $this->post_option_name_list );
+        $this->taxonomy_option_name = $this->get_option_name( $this->taxonomy_option_name_list );
+
+        $this->post_types = (array) get_option( $this->post_option_name, array() );
```

Comment:
We should check if we found a valid option name before doing this and the next line.